modules/test/playwright/pages/commerce/commerceMiniCartPage.ts

```diff
@@ -44,6 +49,7 @@ export class CommerceMiniCartPage {
 		this.quickAddToCartButton = page.getByTestId(
 			'quick-add-to-cart-button'
 		);
+		this.quickAddToCartSku = page.getByRole('menuitem', {name: this._sku});
```

Comment:
This should be a function that accepts the sku as parameter and returns the locator